homeassistant/components/zwave_js/api.py

```diff
@@ -106,6 +114,123 @@
 # constants for migration
 DRY_RUN = "dry_run"
 
+# constants for inclusion
+INCLUSION_STRATEGY = "inclusion_strategy"
+PIN = "pin"
+FORCE_SECURITY = "force_security"
+PLANNED_PROVISIONING_ENTRY = "planned_provisioning_entry"
+QR_PROVISIONING_INFORMATION = "qr_provisioning_information"
+QR_CODE_STRING = "qr_code_string"
+
+DSK = "dsk"
+
+VERSION = "version"
+GENERIC_DEVICE_CLASS = "generic_device_class"
+SPECIFIC_DEVICE_CLASS = "specific_device_class"
+INSTALLER_ICON_TYPE = "installer_icon_type"
+MANUFACTURER_ID = "manufacturer_id"
+PRODUCT_TYPE = "product_type"
+PRODUCT_ID = "product_id"
+APPLICATION_VERSION = "application_version"
+MAX_INCLUSION_REQUEST_INTERVAL = "max_inclusion_request_interval"
+UUID = "uuid"
+SUPPORTED_PROTOCOLS = "supported_protocols"
+
+FEATURE = "feature"
+UNPROVISION = "unprovision"
+
+# https://github.com/zwave-js/node-zwave-js/blob/master/packages/core/src/security/QR.ts#L41
+MINIMUM_QR_STRING_LENGTH = 52
+
+
+def handle_planned_provisioning_entry(info: dict) -> ProvisioningEntry:
```

Comment:
handle is a weird choice. Maybe `convert`?

Reply:
I prefer a static method like `ProvisioningEntry.from_dict(info)`

Reply:
The reason we can't use `ProvisioningEntry.from_dict` here is that converts the data from the driver (camel case) vs in HA's WS API we use snake style for the inputs. If we switched to use camel case, we could just use this helper function, but we've gone back and forth about this in the past and I believe last time we decided HA should be as consistent as possible, e.g. snake style

Reply:
`from_ws_dict` 😝 

Can we then maybe call this `provisioning_entry_from_ws_msg` ?